templates/zerver/help/invite-new-users.md

```diff
@@ -1,162 +1,90 @@
-
-1. Set **Restrict email domains of new users?** to
-   **Restrict to a list of domains**.
-
-1. Enter any number of domains. For each domain, check or uncheck
-   **Allow subdomains**.
-
-1. Click **Save changes**.
+{relative|gear|invite}
 
-{tab|allow-anyone-to-join}
+1. Enter a list of email addresses.
 
-1. Set [default streams](/help/set-default-streams-for-new-users) for new users.
+1. Select when the invitation will expire.
 
-{settings_tab|organization-permissions}
+1. Select what [role](/help/roles-and-permissions) the users will join as.
 
-1. Find the section **Joining the organization**.
+1. Configure which streams they will be added to.
 
-1. Toggle **Invitations are required for joining this organization**.
+1. Click **Invite**.
 
-1. Set **Restrict email domains of new users?** to either
-   **Don't allow disposable email addresses** (recommended)
-   or **No restrictions**.
+!!! warn ""
 
-1. Click **Save changes**.
+    **Note**: The number of email invites you can send in a day is
+    limited on the Zulip Cloud Free plan. [Contact support](/help/contact-support)
+    if you hit the limit and want to invite more users.
 
 {end_tabs}
 
-Before anyone joins your organization this way, we'll send a validation link
-to verify their email address.
-
-## Send invitations
+## Send an invitation link
 
-By default, organization admins and members can send
-invitations. Organization admins can also change who can send invitations.
-
-Note that on most Zulip servers (including Zulip Cloud), email invitations
-and reusable invitation links expire 10 days after they are sent.
+{!admin-only.md!}
 
 {start_tabs}
 
-{tab|send-email-invitations}
-
-{relative|gear|invite}
-
-1. Enter a list of email addresses.
-
-1. Decide whether the users should join as [owners, admins, moderators,
-   members, or guests](/help/roles-and-permissions).
-
-1. Select which streams they should join. If you send invitations often, you
-   may want to configure a set of
-   [default streams](/help/set-default-streams-for-new-users).
-
-1. Click **Invite**.
-
-!!! warn ""
-
-    * You will only see **Invite users** in the gear menu if you have
-    permission to invite users.
-    * The number of email invites you can send in a day is limited in
-    the free plan. [Contact us](/help/contact-support) if you hit the
-    limit and want to invite more users.
-
-{tab|share-an-invite-link}
-
 {relative|gear|invite}
 
 1. Click **Generate invite link**.
 
-1. Decide whether users using the link should join as [owners, admins, moderators
-   members, or guests](/help/roles-and-permissions).
+1. Select when the invitation will expire.
+
+1. Select what [role](/help/roles-and-permissions) the users will join as.
 
-1. Select which streams they should join. If you send invitations often, you
-   may want to configure a set of
-   [default streams](/help/set-default-streams-for-new-users).
+1. Configure which streams they will be added to.
```

Comment:
I'm not sure I have a better proposal, but I kinda feel like "which streams they will be initially subscribed to" would better express the role of these streams. In particular, that would reduce the potential confusion as to whether these will be in addition to "default streams". (Default streams are ignored if streams were specified in creating an invitation).

Reply:
That change works for me.

Reply:
I decided to drop this since we've addressed it in the summary at the top.